risc0/zkvm/src/receipt.rs

```diff
@@ -13,6 +13,55 @@
 // limitations under the License.
 
 //! Manages the output and cryptographic data for a proven computation
+//!
+//! # Usage
+//! Receipts are typically constructed as the output of the prover
+//! ```ignore
+//! // In real code, the ELF & Image ID would be generated by risc0 build scripts from guest code
+//! use methods::{EXAMPLE_ELF, EXAMPLE_ID};
+//! use risc0_zkvm::Prover;
+//!
+//! let mut prover = Prover::new(&EXAMPLE_ELF, EXAMPLE_ID)
+//!     .expect("Prover should be constructed from an ELF file and matching Image ID");
+//! let receipt = prover
+//!     .run()
+//!     .expect("Prover should run guest code that doesn't crash to generate a receipt");
```

Comment:
I don't think "crash" is a good term to use, especially in Rust. There are many reasons this could fail, for instance the embedded verification could fail after constructed a proof. Or there could be a constraint failure in the circuit, or perhaps the guest actually did panic.